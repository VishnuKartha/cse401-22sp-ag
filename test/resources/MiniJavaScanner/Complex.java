public int search(){
    int i = 0;
    int[] arr = new int[10];
    while(i < arr.length){
        if(arr[i] < 0){
            i++;
        }else{
            i--;
        }
    }
    return i;
}