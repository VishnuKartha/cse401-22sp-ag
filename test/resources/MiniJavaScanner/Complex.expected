PUBLIC INT ID(search) LPAREN RPAREN LBRACE INT ID(i) BECOMES INTEGER_LITERAL(0) SEMICOLON INT LBRACKET RBRACKET ID(arr) BECOMES NEW INT LBRACKET INTEGER_LITERAL(10) RBRACKET SEMICOLON WHILE LPAREN ID(i) LESS_THAN ID(arr) PERIOD LENGTH RPAREN LBRACE IF LPAREN ID(arr) LBRACKET ID(i) RBRACKET LESS_THAN INTEGER_LITERAL(0) RPAREN LBRACE ID(i) PLUS PLUS SEMICOLON RBRACE ELSE LBRACE ID(i) MINUS MINUS SEMICOLON RBRACE RBRACE RETURN ID(i) SEMICOLON RBRACE